<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QWidget" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add the title of the Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add the Description of the Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add the Price of the Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add the Images for the Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AddImage">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>AddImage</string>
   </property>
  </widget>
  <widget class="QFrame" name="Pathframe">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add the Quantity of the Product:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQuantity">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>510</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>650</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
